rustfmt/action.yml

```
@@ -22,8 +22,21 @@ runs:
     - name: Rustfmt Job Summary
       shell: bash
       run: |
+        # Find Cargo config file
+        declare -a config_file_locations=(".cargo/config.toml" ".cargo/config")
+        config_file=""
+        for file in "${config_file_locations[@]}"; do
+          if [ -f "$file" ]; then
+            config_file="$file"
+            break
+          fi
+        done
+        if [ -z "$config_file" ]; then
+          echo "No config file found!";
```

if i understand the intent here, it's probably a good idea to exit with a failure if the config_file isn't found, rather than an error in the `alias` line. 
```suggestion
          echo "No config file found!";
          exit 1;
```